pkg/rpcservice/rpc_service.go

```diff
@@ -34,6 +35,9 @@ type RPCService struct {
 	cfg *Config
 
 	grpcServer *grpc.Server
+
+	// TelemetryClient to use for sending telemetry events
+	TelemetryClient stripe.TelemetryClient
```

Comment:
why is `TelemetryClient` capitalized when other class attrs are not?

Reply:
This is for making the `TelemetryClient` accessible publicly for the interceptor right?

Reply:
Yup, that's correct!